Workbooks/Workloads-AMA/Azure SQL Managed Instance/Azure SQL Managed Instance.workbook

```diff
@@ -295,39 +295,11 @@
         "version": "NotebookGroup/1.0",
         "groupType": "editable",
         "items": [
-          {
-            "type": 9,
-            "content": {
-              "version": "KqlParameterItem/1.0",
-              "parameters": [
-                {
-                  "id": "1645978b-604a-41b1-b7e3-071208af0ec8",
-                  "version": "KqlParameterItem/1.0",
-                  "name": "ClerkType",
-                  "label": "Show",
-                  "type": 2,
-                  "isRequired": true,
-                  "value": "Only SQL memory clerks",
-                  "typeSettings": {
-                    "additionalResourceOptions": []
-                  },
-                  "jsonData": "[\n    \"All memory clerks\",\n    \"Only SQL memory clerks\"\n]",
-                  "timeContext": {
-                    "durationMs": 86400000
-                  }
-                }
-              ],
-              "style": "above",
-              "queryType": 0,
-              "resourceType": "microsoft.operationalinsights/workspaces"
-            },
-            "name": "parameters - 1"
-          },
           {
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "let data = InsightsMetrics\r\n| where Namespace == 'sqlserver_memory_clerks'\r\n| extend Tags = todynamic(Tags)\r\n| extend Database = tostring(Tags.database_name), Instance = tostring(Tags.sql_instance), ClerkType = tostring(Tags.clerk_type)\r\n| where (Computer == '{Computer}' or '*' == '{Computer}') and Instance == '{SqlInstance}' and (Database == '{SqlDatabase}' or '*' == '{SqlDatabase}') and (('{ClerkType}' == 'Only SQL memory clerks' and ClerkType startswith 'SQL') or '{ClerkType}' == 'All memory clerks');\r\ndata\r\n| extend SeriesName = strcat(Instance, '/', Database)\r\n| summarize (StartTime, StartValue) = arg_min(TimeGenerated, Val), (EndTime, EndValue) = arg_max(TimeGenerated, Val) by bin(TimeGenerated, {TimeRange:grain}), SeriesName, ClerkType\r\n| extend Val = iff(EndValue >= StartValue, EndValue - StartValue, real(null))\r\n| make-series Trend = sum(Val) default = 0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by ClerkType, SeriesName\r\n\r\n",
```

Comment:
removed ` and (('{ClerkType}' == 'Only SQL memory clerks' and ClerkType startswith 'SQL') or '{ClerkType}' == 'All memory clerks')`